java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -4737,11 +4781,10 @@ public void xpending(BaseClient client) {
                                 InfRangeBound.MIN,
                                 InfRangeBound.MAX,
                                 10L,
-                                StreamPendingOptions.builder().minIdleTime(1L).consumer(consumer2).build())
+                                StreamPendingOptions.builder().minIdleTime(1L).consumer(consumer1).build())
                         .get();
-        assertEquals(1, pending_results_extended.length);
-        assertEquals(streamid_5, pending_results_extended[0][0]);
-        assertEquals(consumer2, pending_results_extended[0][1]);
+        // note: consumer claimed all remaining messages
```

Comment:
not sure I understand the comment, did you mean this?
```suggestion
        // note: streams ID 1 and 5 are still pending, all others were acknowledged
```